### DISCUSSION TITLE
tutorials/scripting/c_sharp/c_sharp_signals

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#6] [User-contributed notes]
# tutorials/scripting/c_sharp/c_sharp_signals

For a detailed explanation of signals in general, see the Using signals section in the step by step tutorial. While it is still possible to use signals through the Connect/ Disconnect API, C# gives...

https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/c_sharp_signals.html

**Comment by 31** [comment]
[Moved to [Elaborate on C# Connect/Disconnect #8812](https://github.com/godotengine/godot-docs/pull/8812) after rule change.]
